addons/planner/static/src/js/planner.js

```diff
@@ -288,6 +289,9 @@ var PlannerDialog = Widget.extend({
         }
         self.update_planner(page_id);
     },
+    on_close:function(){
+            this.trigger("hide.bs.modal");
```

Comment:
rebase your branch and remove this change. Planner has change, there is another way to do that ;)